src/views/Submit.vue

```diff
@@ -315,6 +346,22 @@ export default {
 			// Using button-click event to not bypass validity-checks and use our specified behaviour
 			this.$refs.submitButton.click()
 		},
+		addFormFieldToLocalStorage(question) {
+			if (!this.isLoggedIn) {
+				return
+			}
+			this.formValuesForLocalStorage[`${question.id}`] = { value: this.answers[question.id], type: answerTypes[question.type].component.name }
+			const parsed = JSON.stringify(this.formValuesForLocalStorage)
+			localStorage.setItem(`nextcloud_forms_${this.publicView ? this.shareHash : this.hash}`, parsed)
+		},
+		deleteFormFieldFromLocalStorage() {
+			if (!this.isLoggedIn) {
+				return
+			}
+			this.formValuesForLocalStorage = {}
+			const parsed = JSON.stringify(this.formValuesForLocalStorage)
+			localStorage.setItem(`nextcloud_forms_${this.publicView ? this.shareHash : this.hash}`, parsed)
```

Comment:
```suggestion
			localStorage.removeItem(`nextcloud_forms_${this.publicView ? this.shareHash : this.hash}`, parsed)
```
Is there a reason why you just set the item to an empty object and don't remove the item completely?

Reply:
I can't think of a reason why I did that haha, fixed :))